.github/workflows/ci.yml

```diff
@@ -140,20 +140,26 @@ jobs:
       # Generate a unique object ID prefix to mark the files in GCS and BigQuery so that we run multiple
       # CI simultaneously.
       - name: 'Generate object prefix'
+        id: 'generate_object_prefix'
         run: |
-          echo "INTEG_TEST_OBJECT_PREFIX=$RANDOM" >> $GITHUB_ENV
+          echo "INTEG_TEST_OBJECT_PREFIX=${RANDOM}" >> "$GITHUB_ENV"
+      - name: 'Output object prefix'
```

Comment:
Can you combine generate and output object prefix in one step since each step is just one echo?

Reply:
This was one of Suhong's suggestion, and was my first approach, it was a bit problematic, so I separated it.

Reply:
Just curious, what was the problem when you combine them in one step?

Reply:
when using `${{needs.xxxx.INTEG_TEST_OBJECT_PREFIX}}` in the following step, the value is empty.

Reply:
By the way, looks like you are using INTEG_TEST_OBJECT_PREFIX and INTEG_TEST_OBJECT_PREFIX_OUTPUT with same value. Maybe you syntax has something wrong.
Does the following work?
1. define `OBJECT_PREFIX` variable = {RANDOM}
2. echo "INTEG_TEST_OBJECT_PREFIX=${OBJECT_PREFIX}" >> "$GITHUB_ENV"
3. echo "INTEG_TEST_OBJECT_PREFIX_OUTPUT=${OBJECT_PREFIX}" >> "$GITHUB_OUTPUT"